lib/krikri/aggregation_entity_behavior.rb

```diff
@@ -0,0 +1,14 @@
+
+# TODO:  move to a subdirectory?
```

Comment:
Maybe `lib/krikri/entity_behaviors`, as we've done with something like parsers? (see [here](https://github.com/dpla/KriKri/blob/develop/lib/krikri/parsers/json_parser.rb))